### DISCUSSION TITLE
`Config/` fields in `DESCRIPTION`

### DISCUSSION THREAD

**Discussion by Bisaloo** [epiverse-trace/epiverse-trace.github.io#328] [Blog post ideas]
Prompted by a discussion with @TimTaylor on slack. As far as we know, there is no central place documenting all existing `Config/` fields

Here are some `Config/` I know:

- `Config/Needs/*` in pak
- `Config/testthat/edition`, `Config/testthat/parallel`, `Config/testthat/start-first` in testthat
- `Config/usethis/last-upkeep` by usethis
- to be completed

**Comment by jamesmbaazam** [comment]
`Config/Needs/website`
